src/components/Pages/ActionsPage/ActionsPage.js

```diff
@@ -268,9 +270,14 @@ class ActionsPage extends React.Component {
         disableOverlayClose: true,
       },
     ];
-
     return (
       <div id="test-actions-main-wrapper">
+        {Seo({
+          title: "Actions",
+          description: "",
+          url: `${window.location.pathname}`,
+          subdomain: community?.community?.subdomain,
```

Comment:
this doesn't look right

Reply:
the whole thing or only that block of code ?